frontend/src/app/components/clickable_card.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Link from "next/link";
+
+interface ClickableCardProps {
+  href: string;
+  imageSrc: string;
+  title: string;
+}
+
+interface ClickableCardListProps {
+  items: ClickableCardProps[];
+}
+
+const ClickableCard: React.FC<ClickableCardProps> = ({
```

Comment:
It's fine, but creating a component for each file is always recommended to avoid overcomplicating and creating long TSX files. 
It's okay to have two components in the same file (because they are both card components), but if you plan to add more in the future, consider splitting them into separate files.

Reply:
sure!